python/ray/tune/trainable.py

```diff
@@ -320,7 +318,7 @@ def save(self, checkpoint_dir=None):
             raise ValueError("Returned unexpected type {}. "
                              "Expected str or dict.".format(type(checkpoint)))
 
-        with open(checkpoint_path + ".tune_metadata", "wb") as f:
+        with open(os.path.join(checkpoint_dir, ".tune_metadata"), "wb") as f:
```

Comment:
why change this? Doesn't this break backwards compat?

Reply:
I think you need to revert this - this breaks restoring Trainable for people using older versions of Tune.